pallets/network-score/src/tests.rs

```diff
@@ -159,3 +159,89 @@ fn check_duplicate_message_id() {
 		);
 	});
 }
+
+#[test]
+fn check_entity_mismatch() {
```

Comment:
nit: Please change to `revise_rating_with_entry_entity_mismatch_should_fail`

Reply:
Done the required changes